app/assets/javascripts/models/data/granule.coffee

```diff
@@ -37,7 +37,8 @@
       return time_start unless time_end?
       return time_end unless time_start?
 
-      "<p class=\"temporal-start\">#{time_start}</p><p class=\"temporal-end\">#{time_end}</p>"
+#      "<p class=\"temporal-start\"><h5 class=\"temporal-start-label\">START</h5>#{time_start}</p><p class=\"temporal-end\"><h5 class=\"temporal-end-label\">END</h5>#{time_end}</p>"
```

Comment:
You probably just commented this out while you were experimenting, just make sure to delete the line when you're done.